scvi/module/_vae.py

```diff
@@ -490,7 +554,6 @@ def loss(
         return LossOutput(
             loss=loss, reconstruction_loss=reconst_loss, kl_local=kl_local
```

Comment:
Add MMD loss to LossOutput so that it is available in `model.history`

Reply:
Might need to go through `LossOutput` code in `scvi.module.base._base_module` and add changes there too.